content/posts/newsletter-015/index.md

```diff
@@ -668,6 +668,47 @@ on the module's [repository][Proton-Media-Converter-Github]
 [Proton-Github]: https://github.com/ValveSoftware/Proton
 [Proton-Media-Converter-Github]: https://github.com/ValveSoftware/Proton/tree/proton_5.13/media-converter
 
+### [Rust GPU][RustGPU-Github] v0.1
+
+![Rust GPU Sky](rustgpu.jpg)
+_Sky example in Rust GPU_
+
+Rust GPU is a very early stage project to make Rust a first-class language and
+ecosystem for building GPU code.
+
+[Rust GPU released v0.1 in October][RustGPU-Twitter], and has already garnered
+over 2000 stars on Github. There are also threads about it [on Hacker
+News][RustGPU-HackerNews] and [Reddit][RustGPU-Reddit]. Currently, compiling and
+running very simple shaders works, and a significant portion of the core library
+also compiles. However, many things aren't implemented yet: for example, loops
+and switches aren't supported yet! That means that while being technically
+usable, Rust GPU is far from being production-ready.
+
+Here is a description of the project:
+
+```
```

Comment:
This should probably be marked up as a quote (i.e. start each line with `>`) rather than as a fenced code block.

Also, the CI is currently complaining because this block doesn't have a language at the start (e.g. `rust`), so making that change should also make the build pass.

Reply:
:1st_place_medal: